pages/leagues/[id].tsx

```diff
@@ -19,11 +21,23 @@ interface TeamProps{
 
 
 const League:React.FC<TeamProps> =({teams}) => {
+  
+  async function deleteLeague(){
+
+    try {
+      await http.delete(`/leagues/${router.query.id}`);
```

Comment:
esse endpoint tá errado é bom testar excluir uma liga para verificar se está funcionando, precisar perguntar a @Msouza-95 qual a rota correta

Reply:
Beleza confirmei com ele, pela descrição que ele me passou parece que é assim mesmo liga e o id da liga, esse router.query.id fica com o id da liga, só se for pra pegar o id da liga de outro jeito, sendo desse jeito errado